src/System.Net.Sockets/tests/FunctionalTests/DualModeSocketTest.cs

```diff
@@ -376,6 +376,22 @@ public void Socket_SendToAsyncV4IPEndPointToV4Host_Throws()
             Assert.Equal(SocketError.Fault, args.SocketError);
         }
 
+        // NOTE: on *nix, this API returns ENETUNREACH instead of EFAULT: these platforms
+        //       check the family of the provided socket address before checking its size
+        //       (as long as the socket address is large enough to store an address family).
+        [Fact] // Base case
+        [PlatformSpecific(PlatformID.Unix)]
+        public void Socket_SendToAsyncV4IPEndPointToV4Host_Throws_Unix()
```

Comment:
Not directly related to the change, but is the name of this test (and the one it copies from) correct?  We're creating a v6 socket, trying to use a v4 endpoint, but the test is named v4 to v4 rather than v4 to v6?  I'm probably just not understanding the meaning.